vscode-trace-extension/src/trace-explorer/opened-traces/trace-explorer-opened-traces-webview-provider.ts

```diff
@@ -59,6 +59,10 @@ export class TraceExplorerOpenedTracesViewProvider implements vscode.WebviewView
         }
     }
 
+    public rerender() {
```

Comment:
I think you can call the lines below in method `updateTraceServerUrl()`. With this you can avoid adding this public API.